src/Api/Client/APIClient.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
```

Comment:
This blank line does not go, remove it. Review [PSR-2](http://www.php-fig.org/psr/psr-2/)